src/graphql/root/query/me.ts

```diff
@@ -1,17 +1,23 @@
+import {
+  addAttributesToCurrentSpanAndPropagate,
+  SemanticAttributes,
+  ENDUSER_ALIAS,
+} from "@services/tracing"
 import { GT } from "@graphql/index"
 
 import { UserWithAccounts } from "@graphql/types/object/user"
-import * as Users from "@app/users"
 
 const MeQuery = GT.Field({
   type: UserWithAccounts,
-  resolve: async (_, __, { uid }) => {
-    const user = await Users.getUser(uid)
-    if (user instanceof Error) {
-      throw user
-    }
-    return user
-  },
+  resolve: async (_, __, { ip, domainUser }) =>
```

Comment:
domainuser is the user from `await Users.getUser(uid)`? 

this one is actually an optimization where prior to this we were calling `getUser` twice?

Reply:
It’s the response from getUserForLogin in the context - used for v2. - Yes it’s an optimization